src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,11 +5,26 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private double dozingWidth;
+
     public Bulldozer() {
     }
 
+    public Bulldozer(double dozingWidth) {
+        this.dozingWidth = dozingWidth;
+    }
+
     @Override
     public void doWork() {
         System.out.println("Bulldozer started to work");
     }
+
+    public double getDozingWidth() {
+        return dozingWidth;
+    }
+
+    public Bulldozer setDozingWidth(int dozingWidth) {
+        this.dozingWidth = dozingWidth;
+        return this;
```

Comment:
The method 'setDozingWidth' should have a 'void' return type instead of returning 'Bulldozer'. If you're aiming for a fluent interface (method chaining), make sure that it is a requirement of your design.